doffer.ts

```diff
@@ -187,14 +187,34 @@ async function getNOPVInfo(pageGetter: PageGetter, bbl: BBL, cache: DOFCache, fi
 
   for (let link of links) {
     const name = `${link.date} NOPV for BBL ${bbl}`;
-    const text = await pageGetter.cachedDownloadAndConvertPDFToText(bbl, link.url, name, cache, `nopv-${link.date}`, ['-layout']);
+    const cacheSubkey = `soa-${link.date}`;
```

Comment:
will changing the subkey format from `nopv-{date}` to `soa-{date}` cause cache misses? same for changing the extra flag from `-layout` to `-table` ?

Reply:
oh wow yes, that's a typo! Thanks so much for catching this!